### ISSUE TITLE
Support alternative image host to avoid new Docker Hub limits

### ISSUE THREAD

**Issue by ofek** [pypy/manylinux#15]
https://www.docker.com/blog/scaling-docker-to-serve-millions-more-developers-network-egress/

https://github.com/pypa/manylinux uses https://quay.io

**Comment by asottile** [comment]
easier is probably to use github's thing (ghcr) -- I've had some success with that [here](https://github.com/pre-commit-ci/runner-image)

I've also seen much better performance out of ghcr than quay, but that may be anecdotal

**Comment by antocuni** [comment]
I agree it would be a good idea to move away from docker hub. PRs are welcome ;)
